aten/src/ATen/native/quantized/cudnn/Conv.cpp

```diff
@@ -72,13 +72,14 @@ cudnn_frontend::ConvDesc_v8 getConvDescriptor(cudnnDataType_t dataType, IntArray
 
 // TODO: there is a table from input dtype to operator dtype, we can derive
 // the operator dtype based on input dtype
-cudnn_frontend::PointWiseDesc_v8 getPointWiseMulDescriptor(cudnnDataType_t dataType) {
+cudnn_frontend::PointWiseDesc_v8 getPointWiseDescriptor(cudnnDataType_t dataType, cudnnPointwiseMode_t pw_op_type) {
```

Comment:
I feel it's still good to keep `getPointWiseMulDescriptor` so that we don't need to pass around `CUDNN_POINTWISE_MUL`, it can call this function.
We can create a `getPointWiseAddDescriptor` as well